cpp/dolfinx/fem/assemble_matrix_impl.h

```diff
@@ -80,6 +85,12 @@ void assemble_cells(
   for (std::size_t index = 0; index < cells.size(); ++index)
   {
     std::int32_t c = cells[index];
+    // Map the cell in the integration domain to the cell in the mesh
+    // each function space is defined over
+    std::int32_t c_0 = cell_map_0(cells.subspan(index, 1));
```

Comment:
What is the point of `subspan` here? Isn't it just the same as `cells[index]`?

Reply:
OK, I see that `cell_map_0` has to take a `span` argument... But if it is always length 1, does that make sense? I guess for facets, it is length 2, so it is consistent...